state/protocol/protocol_state/kvstore/set_value_statemachine_test.go

```diff
@@ -110,5 +110,5 @@ func (s *SetKeyValueStoreValueStateMachineSuite) TestEvolveState_SetEpochExtensi
 func (s *SetKeyValueStoreValueStateMachineSuite) TestBuild() {
 	dbOps, err := s.stateMachine.Build()
 	require.NoError(s.T(), err)
-	require.True(s.T(), dbOps.IsEmpty())
+	require.True(s.T(), len(dbOps) == 0)
```

Comment:
removing object-orientation makes syntax more complicated. This is minor here. Though, overall since we are brining deferredDBOps back, I think we should keep the implementation it in an object-oriented fashion and revert this here.

Reply:
[Yes, I reverted it here](https://github.com/onflow/flow-go/pull/7609/files#diff-4114f49282c1800c5c7ce604c0cbac697e842e7f9371da1fe8e7f9bcb3d7577aR113)